projects/experimental/components/textfield/textfield.template.html

```diff
@@ -0,0 +1,34 @@
+<ng-container *ngIf="directive?.change$ | async"></ng-container>
+<ng-content select="input"></ng-content>
+<ng-content select="select"></ng-content>
+<ng-content select="label"></ng-content>
+<span
+    class="t-content"
+    (waResizeObserver)="side = $event[0].contentRect.width"
+>
+    <ng-content></ng-content>
+    <button
+        *ngIf="options.cleaner"
+        appearance="icon"
+        iconLeft="tuiIconClose"
+        size="xs"
+        tabindex="-1"
+        tuiIconButton
+        class="t-clear"
+        (click)="input!.nativeElement.value = ''"
+        (pointerdown.silent.prevent)="input!.nativeElement.focus()"
```

Comment:
can we override the button close template by PolymorpheusComponent?

```html
<ng-container *ngIf="options.cleaner">
    <ng-container *polymorpheusOutlet="buttonComponent as text; context: {$implicit: input}">
        {{ text }}
    </ng-container>
</ng-container>
```

Reply:
Let's not overcomplicate things. You can always disable built-in cleaner and add your own inside content.